charts/osm/README.md

```diff
@@ -23,6 +23,12 @@ A Helm chart to install the OSM control plane on Kubernetes
 | OpenServiceMesh.deployPrometheus | bool | `false` |  |
 | OpenServiceMesh.enablePrometheusScraping | bool | `true` | Cannot be false if `deployPrometheus` is true |
 | OpenServiceMesh.envoyLogLevel | string | `"error"` |  |
+| OpenServiceMesh.fluentBit.name | string | `"fluentbit-logger"` |  |
+| OpenServiceMesh.fluentBit.registry | string | `"fluent"` |  |
+| OpenServiceMesh.fluentBit.tag | string | `"1.6.4"` |  |
+| OpenServiceMesh.fluentBit.pullPolicy | string | `"IfNotPresent"` |  |
+| OpenServiceMesh.fluentBit.defaultOutput | string | `"stdout"` |  |
+| OpenServiceMesh.fluentBit.allowCustomOutput | bool | `"true"` |  |
```

Comment:
Should this default to `false` since this is only applicable to Log Analytics?

Reply:
Given that `Customer_ID` and `Shared_Key` only make sense if the output plugin is log analytics, I think it's more intuitive to only add these configs when `azure` is specified.  For default values, `fluentBit.outputName` should default to `stdout`. For charts that depend on this chart, they can override this `outputName` and provide `workspaceID` and `primaryKey` accordingly. Also `workspaceID` and `primaryKey` should be in values.schema.json as well.

WDYT?

e.g.
```yaml
    [OUTPUT]
      Name    {{ .Values.OpenServiceMesh.fluentBit.outputName }}
      Match   *
      {{- if eq .Values.OpenServiceMesh.fluentBit.outputName "azure" }}
      Customer_ID {{ .Values.OpenServiceMesh.fluentBit.workspaceID }}
      Shared_Key {{ .Values.OpenServiceMesh.fluentBit.primaryKey }}
      {{- end }}
```

Reply:
Oh yes this sounds a lot more intuitive! I think that should work well

Reply:
Addressed in follow up PR: https://github.com/openservicemesh/osm/pull/2116